eduroam.md

```diff
@@ -129,12 +152,15 @@ This file will contain most of these lines already. The `uuid` field will alread
 
 5. Run the following command: 
 ```
-# nmcli conection up eduroam
+# nmcli conection up eduroamWiFi
 ```
+This tells `nmcli` that you want to use the configuration `eduroamWiFi`, which was edited in the previous step. 
 6. (Optional) Test your connection by running: 
 ```
 $ ping 9.9.9.9
 ```
+This command tests the ability to talk to the [Quad9 DNS Service](https://www.quad9.net/). If the command returns an error, then you are not connected to the `eduroam` network (or any other network, for that matter). 
+
 ---
 The following configurations must use the listed file types. Use of alternative file types has not been tested, and is not recommended.
 
```

Comment:
this seems a bit strange to have a horizontal rule here and then the next heading for tthe instructions that follow.

i wonder if making each set of instructions its own self contained thing containing these sections might be better:
- notes/quirks/warnings/assumptions (stick to these file types, run commands as root .etc)
- requirements/what you need
- the instructions

then each set is self contained and probably easier to split into another page

Reply:
I plan on making a unique page for each method in the future, I wanted to make sure the current active page is up-to-date first.

Reply:
nah this is fine, especially considering ITS now links to it